app/models/session.rb

```diff
@@ -209,6 +209,9 @@ def as_json(opts=nil)
       end
     end
 
+    first_measurement = fetch_first_measurement_coordinates
+    res.merge!(:starting_longitude => first_measurement.longitude, :starting_latitude => first_measurement.latitude)
+
```

Comment:
this file is work in progress, I wanted to focus on js first

Reply:
👍